src/Heatmap/GMUHeatmapTileLayer.m

```diff
@@ -24,318 +24,331 @@
 
 static const int kGMUTileSize = 512;
 static const int kGMUMaxZoom = 22;
-static const int kGMUMinZoomIntensity = 5;
-static const int kGMUMaxZoomIntensity = 10;
 
 static void FreeDataProviderData(void *info, const void *data, size_t size) { free((void *)data); }
 
 // Holder for data which must be consistent when accessed from tile creation threads.
 @interface GMUHeatmapTileCreationData : NSObject {
- @public
-  GQTPointQuadTree *_quadTree;
-  GQTBounds _bounds;
-  NSUInteger _radius;
-  NSArray<UIColor *> *_colorMap;
-  NSArray<NSNumber *> *_maxIntensities;
-  NSArray<NSNumber *> *_kernel;
+@public
+    GQTPointQuadTree *_quadTree;
+    GQTBounds _bounds;
+    NSUInteger _radius;
+    NSUInteger _minimumZoomIntensity;
+    NSUInteger _maximumZoomIntensity;
+    NSArray<UIColor *> *_colorMap;
+    NSArray<NSNumber *> *_maxIntensities;
+    NSArray<NSNumber *> *_kernel;
 }
 @end
 
 @implementation GMUHeatmapTileCreationData
 @end
 
 @implementation GMUHeatmapTileLayer {
-  BOOL _dirty;
-  GMUHeatmapTileCreationData *_data;
+    BOOL _dirty;
```

Comment:
I'm curious about this whitespace. It looks like you have re-formatted the majority of this file away from Google style. 

/cc @dion-loetscher any thoughts on adding a clang format to this project?

Reply:
@domesticmouse I think adding a clang format is a good idea. 

I think we should hold off on approving this PR until we have added clang format. Then we can revisit this PR after it's been clang formatted.